### ISSUE TITLE
Is this project still under original ownership?

### ISSUE THREAD

**Issue by brundonsmith** [ankit/stylebot#497]
I recently read this article: http://www.zdnet.com/article/firms-buy-popular-chrome-extensions-to-inject-malware-ads/

tl;dr moderately-popular browser extensions are sometimes bought up by companies that want to use them as backdoors to inject anything from ads to malware into users' pages. Given that this one is moderately-but-not-immenseley popular and stylebot.me is no longer online, it seems like a potential target of this sort of thing.

Is there a way to verify that the source is still what's reflected on GitHub, or does the author want to weigh in and verify the extension's proper ownership?

**Comment by brundonsmith** [comment]
For those curious, I was able to find the installed extension under C:\Users\<MY-USER>\AppData\Local\Google\Chrome\User Data\<CHROME-PROFILE>\Extensions\oiaejidbmkiecgbjeifoejpgmdaleoha\2.2_0

I then checked out the github copy, then copied and pasted all of the contents of the installation directory into the local repo, and used git diff to compare the two. The only differences were in manifest.json and some .png files, so I think we're safe at least for now. It would still be good to hear from the author, though, for future purposes.
